### PR TITLE
Defaults fixes + add logger

### PR DESCRIPTION
**Pull request by askmeegs**
Fixes #77

### CODE REVIEW

File: cli/samples/default-config.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# ------ DO NOT EDIT ------------------
+# These are the default values used for `gkekitctl create` when 
+# the user does not supply their own config.yml. Note that the user 
+# *does* have to provide their GCP project ID at runtime.  
+terraformState: local  
+clustersProjectId: "my-project"
+governanceProjectId: "my-project"
+region: "us-central1x"
```
Comment (knee-berts) [inline cli/samples/default-config.yaml:8]: Is that x a typo?

**Review by knee-berts — CHANGES_REQUESTED** [review]
Looks like there is just one typo, aside from that, LGMT

File: cli/samples/default-config.yaml (hunk 1 above)
Comment (askmeegs) [reply cli/samples/default-config.yaml:8]: yes I was testing the validation, will fix!

Comment (knee-berts) [reply cli/samples/default-config.yaml]: Awesome! Thanks Megan!

**Pull request merged**